### DISCUSSION TITLE
ConnMaxIdleTime comment is misleading

### DISCUSSION THREAD

**Discussion by Hagbarth** [redis/go-redis#2270] [General]
Hi!

The doc comment on the `ConnMaxIdleTime` field on `Options` says that the default timeout is 5 minutes. In reality it's being set to 30 minutes further down in the same file.

Happy to open a PR to fix, but I wanted to check if you really want the default to be 30 minutes?

**Comment by vmihailenco** [comment]
Not sure what the value should be, but I've update the comment. Thanks for reporting.

**Comment by recodyx** [comment]
actually I have set `ConnMaxIdleTime` to `5 * time.Minute` but the connections is after 1h still available:

```
 "connectionStats": {
            "total": 128,
            "idle": 128,
            "stale": 391
 }
```
